src/EditorFeatures/CSharpTest/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -1831,11 +1821,7 @@ public async Task FieldIsRead_PartialClass_DifferentFile()
     public int M() => _goo;
 }";
 
-            await new VerifyCS.Test
-            {
-                TestState = { Sources = { source1, source2 } },
-                FixedState = { Sources = { source1, source2 } },
-            }.RunAsync();
+            await VerifyCS.VerifyCodeFixAsync(sources: (source1, source2), fixedSources: (source1, source2), numberOfFixAllIterations: 0);
```

Comment:
> numberOfFixAllIterations: 0

This is the default.

Reply:
The test framework limitation is https://github.com/dotnet/roslyn-sdk/issues/147

Reply:
I believe we have large number of IDE tests which have multiple inputs as the documents. Given the test framework does not support the XML markup based baselines, I think we should add helpers to enable easily porting such tests.